planner/optimize.go

```diff
@@ -214,6 +214,9 @@ func getBindRecord(ctx sessionctx.Context, stmt ast.StmtNode) (*bindinfo.BindRec
 		return nil, ""
 	}
 	globalHandle := domain.GetDomain(ctx).BindHandle()
+	if globalHandle == nil {
```

Comment:
When would this happen?

Reply:
When TiDB starts, we may use the domain's handler(in `LoadPrivilegeLoop`) before initializing it (in `LoadBindLoop`).
![image](https://user-images.githubusercontent.com/7499936/77735872-453f0100-7046-11ea-8472-c562877edd12.png)

Reply:
PTAL @eurekaka